src/serac/physics/utilities/functional/tests/functional_comparisons_cuda.cu

```diff
@@ -0,0 +1,446 @@
+// Copyright (c) 2019-2021, Lawrence Livermore National Security, LLC and
+// other Serac Project Developers. See the top-level LICENSE file for
+// details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+#include <fstream>
+#include <iostream>
+
+#include "mfem.hpp"
+
+#include "axom/slic/core/SimpleLogger.hpp"
+#include "serac/infrastructure/input.hpp"
+#include "serac/serac_config.hpp"
+#include "serac/numerics/expr_template_ops.hpp"
+#include "serac/physics/operators/stdfunction_operator.hpp"
+#include "serac/physics/utilities/functional/functional.hpp"
+#include "serac/physics/utilities/functional/tensor.hpp"
+#include "serac/infrastructure/profiling.hpp"
+#include <gtest/gtest.h>
```

Comment:
Minor: https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

In particular, the gtest header should be higher up